runtime/src/accounts_index.rs

```diff
@@ -48,7 +48,7 @@ pub type AccountMap<V> = InMemAccountsIndex<V>;
 
 pub(crate) type AccountMapEntry<T> = Arc<AccountMapEntryInner<T>>;
 
-pub trait IsCached: 'static + Clone + Debug + PartialEq + ZeroLamport + Copy + Default {
+pub trait IsCached: 'static + Clone + Debug + PartialEq + ZeroLamport + Copy + Default + Sync + Send {
```

Comment:
This trait has grown quite a bit. Maybe in the next round `IsCached` can go back to just the `is_cached()` function, and a new trait can be added for this composite?

Reply:
sure, seems reasonable